doc/JSON_FLAGS.md

```diff
@@ -1003,7 +1003,10 @@ Also see `monster_attacks.json` for more special attacks, for example, impale an
 - ```ACID_BARF``` Barfs corroding, blinding acid.
 - ```ACID``` Spit acid.
 - ```ANTQUEEN``` Hatches/grows: `egg > ant > soldier`.
+- ```BIO_OP_BIOJUTSU``` Attack with a random special martial art maneuverer.
 - ```BIO_OP_TAKEDOWN``` Attack with special martial art takedown maneuverer.
+- ```BIO_OP_DISARM``` Attack with a special martial art disarm maneuverer.
+- ```BIO_OP_IMPALE``` Attack with a strong martial art maneuverer.
```

Comment:
Could you please change `maneuverer` to `maneuver`?

Reply:
Yeah, no problem. That's what I get for copying and pasting!